### PR TITLE
174 sub manager ui

### PR DESCRIPTION
**Pull request by jimmykodes**

### CODE REVIEW

File: src/static/riot/competitions/detail/submission_modal.tag [hunk 1]
```diff
@@ -1,177 +1,125 @@
...
-                    </div>
-                    <div class="item">
-                        <a href="#">Private output from scoring step</a>
-                    </div>
-                </div>
+            <div class="ui fifteen wide column">
+                <div class="ui horizontal divider"></div>
+                <table class="ui very basic table" id="downloads">
+                    <thead>
+                    <tr>
```
Comment (ckcollab) [inline src/static/riot/competitions/detail/submission_modal.tag]: what do you think about indentation with `<thead>`s and such? Should we indent this style, or with an extra indent for `<tr>`s?

File: src/static/riot/competitions/detail/submission_modal.tag [hunk 2]
```diff
@@ -1,177 +1,125 @@
...
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td class="selectable file-download"><a href="{ submission.data_file }"><i class="file archive outline icon"></i> Submission File</a></td>
+                    </tr>
+                    <tr>
+                        <td class="selectable file-download"><a href="{ submission.result }"><i class="file outline icon"></i>Output from prediction step</a></td>
+                    </tr>
+                    <tr>
+                        <td class="selectable file-download"><a href="#"><i class="file outline icon"></i>Output from scoring step</a></td>
```
Comment (ckcollab) [inline src/static/riot/competitions/detail/submission_modal.tag]: do these need to be `a` tags?

File: src/static/riot/competitions/detail/submission_modal.tag [hunk 3]
```diff
@@ -1,177 +1,125 @@
...
-                        <div class="item" data-tab="ingest_stdout">Ingestion stdout</div>
-                        <div class="item" data-tab="ingest_stderr">Ingestion stderr</div>
+                    <div class="ui bottom attached inverted segment tab log" data-tab="p_ingest_stderr">
+                        <pre>{ logs.prediction_ingestion_stderr }</pre>
                     </div>
                 </div>
-                <div class="fluid twelve wide stretched column">
+                <div class="ui tab" data-tab="scoring">
+                    <div class="ui top attached inverted pointing menu">
+                        <div class="active item" data-tab="s_stdout">stdout</div>
```
Comment (ckcollab) [inline src/static/riot/competitions/detail/submission_modal.tag]: how well does switching tabs work with thousands of lines of output?

File: src/static/riot/competitions/detail/submission_modal.tag (hunk 1 above)
Comment (jimmykodes) [reply src/static/riot/competitions/detail/submission_modal.tag]: This styling is what Pycharm defaults to. It does seem a little strange to me. Indenting `<tr>`s makes more sense to me, it just ins't Pycharm's default behavior, so I have left it alone.

File: src/static/riot/competitions/detail/submission_modal.tag (hunk 2 above)
Comment (jimmykodes) [reply src/static/riot/competitions/detail/submission_modal.tag]: Yea, for styling, they need to be a tags

**Pull request merged**